sdk/paloaltonetworks.ngfw/Azure.ResourceManager.PaloAltoNetworks.Ngfw/samples/Generated/Samples/Sample_LocalRulestackResource.cs

```diff
@@ -230,7 +230,7 @@ public async Task Update_LocalRulestacksUpdateMinimumSetGen()
         public async Task Commit_LocalRulestacksCommitMaximumSetGen()
         {
             // Generated from example definition: specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/stable/2023-09-01/examples/LocalRulestacks_commit_MaximumSet_Gen.json
-            // this example is just showing the usage of "LocalRulestacks_Commit" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "LocalRulestacks_commit" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `commit`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/stable/2023-09-01/PaloAltoNetworks.Cloudngfw.json#L499